book/website/community-handbook/governance/gov-constitutional.md

```diff
@@ -0,0 +1,131 @@
+
+### Hypothetical worked example: Collaborating on a funding application
+
+Imagine that there is a funding call looking to support training for open science in India and to build international networks in open science.
+
+The funder is based in India, the money can only be distributed to and managed by an organisation based in India, and the applicants are required to have someone based at an organisation outside of India to be part of the collaboration.
+
+Indrani is a professor at Delhi University in India and Baptiste is a graduate student at Montpellier University in France.
+They have both contributed to The Turing Way as individual contributors at the community decision making level.
+Their idea involves improving resources from The Turing Way and promoting them in their professional networks.
+
+They collaboratively develop their funding proposal with input from the Translation and Localisation working group.
+Before submitting the grant (ideally) they bring a request to the Steering Council to create (if approved) a new delivery group.
+They want to use this funding to strengthen interpersonal and professional connections across The Turing Way, invite new people into the community, and to improve the content for a broader audience.
+
+This Delivery Group would be chaired by Indrani as principle investigator of the project.
+Baptiste would be a member, and Marc - a member of the Turing Way's Translation and Localisation working group - becomes a member of this delivery group in order to maintain strong connections between the T&L Working Group and this delivery group.
+
+Other members would likely include a project manager employed at the University of Delhi, some named collaborators from the grant, and maybe a postdoctoral research fellow who is hired to develop and deliver trainings as described in the grant.
+
+The Turing Way Steering Group would not be accountable for the deliverables in the grant, nor any budget management, nor performance review or appraisals for the team members.
+These responsibilities are external to The Turing Way open source community.
+
+If the Steering Group agree that the proposed work is aligned to the long term strategic vision for the community they would **approve this request**.
+The new delivery group would be required to set up a charter as described in <!-- CHAPTER ON DELIVERY GROUPS --> this chapter's sub-section on delivery groups.
+The chair of the delivery group (Indrani) would become a member of the Steering Council.
+
+If the Steering Group did not agree that the proposed work is aligned to the long term strategic vision for the community they would **reject this request**.
+Indrani and her team could still undertake the work and continue to contribute as individual contributors.
+They can still use materials from The Turing Way under the open source licences that the project uses.
```

Comment:
```suggestion
They can still use materials from *The Turing Way under* the terms of the open source licences that the project uses.
```

Maybe a bit pedantic but the licenses we use may allow distribution as part of a new work under a different license.